deploy/helm/sumologic/values.yaml

```diff
@@ -662,6 +662,8 @@ sumologic:
     ## How many spans per request should be send to receiver
     spans_per_request: 100
 
+    sourceType: http
```

Comment:
shouldn't  this be `otlp` as per readme below?

Reply:
As per what README? Traces use a special HTTP traces source by default, which does accept OTLP, but is supposed to be replaced by our proper OTLP source.